app/views/admin/volunteers/_volunteers_table.html.erb

```diff
@@ -28,7 +26,7 @@
     </tr>
   </thead>
   <tbody>
-    <% volunteers.each do |volunteer| %>
+    <% @volunteers.each do |volunteer| %>
```

Comment:
Shouldn't need to be an instance var if its passed as a local to the partial, right?